config/deploy.rb

```diff
@@ -8,7 +8,7 @@
 set :repo_url, ENV['CAPISTRANO_DEPLOY_REPO_URL'] || 'https://github.com/cantino/huginn.git'
 
 # Default branch is :master
-set :branch, ENV['BRANCH'] || 'master'
+set :branch, ENV['CAPISTRANO_DEPLOY_BRANCH'] || 'master'
```

Comment:
Having the same naming scheme for the variables makes sense, but I think @cantino is right. We should keep `BRANCH` as the second fallback here.

@skddc Can you also change the variable in the [documentation](https://github.com/huginn/huginn/blob/master/doc/manual/capistrano.md#3-run-capistrano)?